frontend/jest.config.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    transform: {
```

Comment:
I think there is some  setting for this file to specify where to look for tests.  Make sure it looks only in the folder for Jest tests

Reply:
Alternatively, you can move the playwright tests somewhere outside the src/ directory and we can reconfigure playwright conf afterwards.